src/main/java/org/mifos/connector/mockPaymentSchema/schema/BatchDetailResponse.java

```diff
@@ -0,0 +1,134 @@
+package org.mifos.connector.mockPaymentSchema.schema;
+
+import java.util.List;
+
+public class BatchDetailResponse {
```

Comment:
do we already have these DTOs in the batch processor/ connector common?
if this is a duplicate can you add them to connector common instead?